src/components/PlayerCardContainer/WinOrLose.js

```diff
@@ -0,0 +1,8 @@
+import React from "react";
+
+function WinOrLose(e) {
```

Comment:
Is this used?  If it is I'm missing where it's being called.

Reply:
I think this was my brain doing overdrive lol, removed the file because atm it's not being used.